modules/ROOT/pages/database-administration/standard-databases/start-stop-databases.adoc

```diff
@@ -0,0 +1,150 @@
+:description: how to start, and stop databases in Neo4j. How to use `START DATABASE` and `STOP DATABASE` Cypher commands.
+
+[role=enterprise-edition not-on-aura]
+= Start and stop databases
+
+Neo4j supports the management of multiple databases within the same DBMS.
+The metadata for these databases, including the associated security model, is maintained in a special database called the `system` database.
+All multi-database administrative commands must be run against the `system` database.
+These administrative commands are automatically routed to the `system` database when connected to the DBMS over Bolt.
+
+[NOTE]
+====
+Administrative commands should not be used during a rolling upgrade.
+For more information, see link:{neo4j-docs-base-uri}/upgrade-migration-guide/upgrade/upgrade-4.4/causal-cluster/[Upgrade and Migration Guide -> Upgrade a cluster].
+====
```

Comment:
I doubt we have this on the drop/alter/composite db pages so it feels odd to have it here. Maybe it should be on the intro page for all the multi-db admin commands instead of on just the start of some of them? 🤔

Like this page: https://neo4j-docs-operations-2372.surge.sh/operations-manual/2025.04/database-administration/